train-student/README.md

```diff
@@ -68,6 +68,25 @@ details.
 - Prepare corpus and adjust variables in `alignment/generate-alignment-and-shortlist.sh`.
 - Generate word alignment and lexical shortlists running the script.
 
+It is recommended to convert the text lexical shortlist to a binary shortlist using `marian-conv` from 
+[Bergamot branch](https://github.com/browsermt/marian-dev/) of marian with `--shortlist,-s` option.
+As the binary shortlist can greatly improve the lexical shortlist loading time (x4k speed-up).
+A usage example of shortlist conversion is as follows:
+```bash
+./marian-conv --shortlist lex.esen.s2t 100 100 0 \    # text shortlist: path first best prune
+               --dump lex.esen.bin \                  # binary shortlist: path
+               --vocabs vocab.esen.spm vocab.esen.spm # source and target vocab path
+```
+The parameters of `100 100 0` are the firstNum, bestNum, and prune threshold
```

Comment:
We normally use `50 50` right?  https://github.com/browsermt/students/commit/f7a0257d54f4d833b50bcfec738a2dbbb97c3ea8#diff-465500ca28ba13efb19951f59ec7bc1f7db85969f5775adfd1cf503b6d483a2c

Reply:
Yes. Let me change to `50 50`.